app/views/layouts/_navbar.html.erb

```diff
@@ -6,6 +6,16 @@
     </button>
     <div class="collapse navbar-collapse" id="navbarSupportedContent">
       <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+        <li class="nav-item">
+          <%= link_to search_path, class: 'nav-link' do %>
+            <i class="fas fa-search"></i> Search
```

Comment:
nit

```suggestion
            <i class="fas fa-search" /> Search
```

Reply:
I tried it and it actually didn't work. Apparently the i-tag shouldn't be self-closing. I found this stack overflow: https://stackoverflow.com/questions/16253890/self-closing-an-i-html-tag-or-not